src/private/MDMPerformerGroup.m

```diff
@@ -72,6 +72,19 @@ - (void)executeLog:(MDMTransactionLog *)log trace:(MDMTrace *)trace {
     if ([performer respondsToSelector:@selector(addPlan:)]) {
       [(id<MDMPlanPerforming>)performer addPlan:plan];
     }
+    
+    // MDMNamedPlanPerforming callbacks
+    if (log.name.length) {
```

Comment:
I'll defer to your better judgement on whether we want to allow `""` as a name.

Not sure I can envisage a reasonable use case for an named plan which has an empty name. Would we not steer people towards the standard `addPlan` method if they didn't want to include a name?